script/nowrap.pl

```diff
@@ -44,6 +45,7 @@
 GetOptions(
     "help" => \&print_usage_and_exit,
     "columns=i" => \$columns,
+    "wrap" => \$isWrap,
```

Comment:
The `wrap --nowrap` invocation is indeed a little weird. I wonder if it would be better to look at `basename($0) eq "wrap"` or a similar trick also/instead?

Reply:
Yes, `nowrap --wrap` first looks contradictory, but then it just means that instead of throwing away the excessive text, I do want it hard-wrapped. I currently don't see much potential for interactive use (as terminals wrap on their own), but I'd probably write an alias (`alias wrap='nowrap --wrap'`) if there were. I don't like tricks with `basename` so much, because they either need a copy step in the Makefile, or a symlink, and the latter often doesn't work with platforms like Cygwin. In any case, the `--wrap` option is a universal and simple solution, and enables users to write their own simple aliases or wrappers if they don't like that option.

Thanks for handling the infinite loop when smaller than `$TAB_STOP`; though it's arguably a stupid value to pass, going into an endless loop surely isn't a nice way to deal with it!